manifests/f/Flakron/ApacheMaven/3.8.9/Flakron.ApacheMaven.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Flakron.ApacheMaven
+PackageVersion: 3.8.9
+InstallerType: zip
+NestedInstallerType: inno
+NestedInstallerFiles:
+- RelativeFilePath: setup.exe
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Oracle.JavaRuntimeEnvironment
Installers:
```

Reply:
Unfortunately it depends on the user what Java SDK they use, it is impossible to know what they need to use and force them to the Oracle one.